recipes/gdbm/all/conanfile.py

```diff
@@ -30,6 +31,7 @@ class GdbmConan(ConanFile):
         "gdbmtool_debug": True,
         "with_libiconv": False,
         "with_readline": False,
+        "with_nls": True,
```

Comment:
If you disable nls, then with_libiconv shouldn't be used as it will no longer look for it.

cf this diff or two builds, both with_libiconv=True, left with `with_nls=True`  and the right `with_nls=False`


![image (4)](https://user-images.githubusercontent.com/5479063/88394806-50215c80-cdc0-11ea-8471-16a6e7453579.png)